app/models/group.js

```diff
@@ -29,8 +27,9 @@ export default class Group extends ModelBase.extend({
   followers     : hasMany('user-follow-group'),
 
   url: computed('identifier', function() {
-    const origin = this.fastboot.isFastBoot ? `${this.fastboot.request.protocol}//${this.fastboot.request.host}` : location.origin;
-    return origin + this.router.urlFor('groups.edit.events', this.id);
+    const origin = this.fastboot.isFastBoot
+      ? `${this.fastboot.request.protocol}//${this.fastboot.request.host}`
+      : location.origin;
+    return origin + `/g/${this.id}`;
```

Comment:
@progmatic-99 
What I see is that only this line is been modified in this PR. Pls revert other indent changes back to normal.
and pls disable any auto-format extention that might have installed in your code-editor to avoid this small indent issues.

Reply:
This is more readable. I will change other things.